core/src/com/unciv/logic/civilization/diplomacy/DiplomacyManager.kt

```diff
@@ -208,8 +208,8 @@ class DiplomacyManager() : IsPartOfGameInfoSerialization {
             val dropPerTurn = getCityStateInfluenceDegrade()
             return when {
                 dropPerTurn == 0f -> 0
-                relationshipLevel() >= RelationshipLevel.Ally -> ceil((getInfluence() - 60f) / dropPerTurn).toInt() + 1
-                relationshipLevel() >= RelationshipLevel.Friend -> ceil((getInfluence() - 30f) / dropPerTurn).toInt() + 1
+                relationshipLevel() >= RelationshipLevel.Ally && 60f > getCityStateInfluenceRestingPoint() -> ceil((getInfluence() - 60f) / dropPerTurn).toInt() + 1
```

Comment:
I think these changes should be in the functions `DiplomacyManager.getCityStateInfluenceDegrade()` instead. The function has the code
```
if (getInfluence() < getCityStateInfluenceRestingPoint())
            return 0f
```
that should probably be changed to `<=` then you wouldn't need the additional conditions in this `when` statement since `getCityStateInfluenceDegrade()` would be 0 and therefore the notifications wouldn't appear.

There's also a similar function `getCityStateInfluenceRecovery()` that could have a similar modification but I don't think that's essential for this PR since that specific value isn't used for notifications.

Reply:
Thanks for pointing this out! Applied these changes and am currently testing them a little more.